src/gpm-control.c

```diff
@@ -387,7 +386,7 @@ gpm_control_hibernate (GpmControl *control, GError **error)
 						       NULL,
 						       &dbus_error );
 		if (proxy == NULL) {
-			egg_error("Error connecting to dbus - %s", dbus_error->message);
+			g_error ("Error connecting to dbus - %s", dbus_error->message);
 			g_error_free (dbus_error);
```

Comment:
And another one.
```
gpm-control.c:390:4: warning: This statement is never executed

                        g_error_free (dbus_error);

                        ^~~~~~~~~~~~
```
https://travis-ci.org/mate-desktop/mate-power-manager/jobs/650313264#L1947
Not sure if these warnings are related to PR, but they are still there.